Sources/GitHub/Model.swift

```diff
@@ -2,19 +2,37 @@ import EmailAddress
 import Foundation
 import Tagged
 
-public struct AccessToken: Codable {
+public struct AccessToken: Codable, Equatable, RawRepresentable {
```

Comment:
-[] separate this into a tagged string (`AccessToken`) and a response payload (`AccessTokenResponse`).